core/lib/mno_enterprise/concerns/models/ability.rb

```diff
@@ -66,86 +66,69 @@ def initialize(user)
     end
 
     #===================================================
-    # Impac
+    # Admin abilities
     #===================================================
-    impac_abilities(user)
+    admin_abilities(user)
```

Comment:
Is the order change intentional?

admin_abilities were called last as there supposed to have precedence over everything:

> An ability rule will override a previous one
See: https://github.com/CanCanCommunity/cancancan/wiki/Ability-Precedence

I guess it doesn't matter in this case as the Impac! ACL is coming from MnoHub anyway?

Reply:
it wasn't intentional - reverting